virkarekisteri/virkarekisteri
# VIR-22 Adding new position from frontend

Added feature to add new position through button in frontend

How to test:
1. Open frontend and run backend
2. Click on the "Uusi virka" button
3. Add details and hit save
4. Confirm that the new post is in the database
